src/App.tsx

```diff
@@ -15,36 +17,98 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  name = 'name',
+  length = 'length',
+}
+
+interface FillterParams {
```

Comment:
There is a typo in the interface name `FillterParams`. It should be `FilterParams` to correctly reflect its purpose and maintain consistency in naming conventions.